src/intrinsics/ecma262/Symbol.js

```diff
@@ -28,11 +28,12 @@ export default function(realm: Realm): NativeFunctionValue {
     let descString;
     if (!description || description instanceof UndefinedValue) {
       descString = undefined;
+    } else if (description instanceof AbstractValue) {
+      return new SymbolValue(realm, description);
```

Comment:
I believe that ToStringPartial expects a concrete value:
return ToString(realm, typeof val === "string" ? val : val.throwIfNotConcrete());